maintenance/aws-janitor/cmd/aws-janitor-boscos/main.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/boskos/client"
+)
+
+var (
+	boskosURL = flag.String("boskos-url", "http://boskos", "Boskos URL")
+	poolSize  = flag.Int("pool-size", 20, "number of concurrent janitor goroutine")
+)
+
+const sleepTime = time.Minute
+
+func main() {
+	flag.Parse()
+
+	logrus.SetFormatter(&logrus.JSONFormatter{})
+	boskos := client.NewClient("AWSJanitor", *boskosURL)
+
+	_ = boskos
+	for {
+
+	}
```

Comment:
an infinite loop without sleep is would occupy a cpu core to it's full capacity.
either `fmt.Scanln` to wait for input instead of the loop or `time.Sleep` inside the loop has to be used.

Reply:
This code isn't intended to be used yet, it's just a skeleton. I'll just comment the for loop out for now.